pkg/rootless/rootless.go

```diff
@@ -90,7 +90,7 @@ func setRootless() error {
 		if err != nil {
 			return parseError
 		}
-		rangeUID, err := strconv.ParseUint(ids[1], 10, 0)
+		rangeUID, err := strconv.ParseUint(ids[2], 10, 0)
```

Comment:
Could you please submit this fix as its own separate commit since it's not directly related to Cloud-Hypervisor support.